app/models/enhancements/artefact.rb

```diff
@@ -85,6 +87,12 @@ def remove_related_artefacts
     end
   end
 
+  def as_json(options={})
+    as_json_original.tap { |hash|
```

Comment:
No, because this is not inheritance; `super` would give the base Mongoid behaviour, rather than the one defined in the original govuk_content_models class.